instance/models/instance.py

```diff
@@ -547,6 +639,19 @@ def studio_url(self):
         """
         return u'{0.protocol}://{0.studio_domain}/'.format(self)
 
+    def update_from_pr(self, pr):
+        """
+        Update this instance with settings from the given pull request
+        """
+        super().update_from_pr(pr)
+        if pr.database_url:
+            self.database_url = pr.database_url
+        if pr.mongo_url:
+            self.mongo_url = pr.mongo_url
+        if pr.ephemeral_databases is not None:
+            self.ephemeral_databases = pr.ephemeral_databases
```

Comment:
This could throw an exception if the user changed this from `True` to `False` without also changing one of the database URLs.